p3exporter/__init__.py

```diff
@@ -41,6 +41,13 @@ def main():
     with open(args.config, 'r') as config_file:
         cfg = yaml.load(config_file, Loader=yaml.SafeLoader)
     collector_config = CollectorConfig(**cfg)
+    if type(cfg.get('logging_levels')) == dict:
```

Comment:
Regarding to codacy results using `is_instance` should be prefered over `type`

Reply:
Changed line to `if isinstance(cfg.get('logging_levels'), dict):` now.